plugins/source/aws/client/client.go

```diff
@@ -88,9 +86,9 @@ func (s *ServicesManager) InitServicesForPartitionAccount(partition, accountId s
 	s.services[partition][accountId].Regions = funk.UniqString(append(s.services[partition][accountId].Regions, svcs.Regions...))
 }
 
-func NewAwsClient(logger zerolog.Logger, b backend.Backend, spec *Spec) Client {
+func NewAwsClient(logger zerolog.Logger, spec *Spec) Client {
 	return Client{
-		Backend: b,
+		// Backend: b,
```

Comment:
If this is getting initialized somewhere else we can probably remove this comment now